# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/canvas/crashtests/crashtests.list
# Commit: 904076df6327
# Full Hash: 904076df6327430699a83cfc75c2caca72dfa872
# Author: Ehsan Akhgari <ehsan@mozilla.com>
# Date: 2019-04-25 22:08:17
# Description:
#   Bug 1546390 - Enable determining whether the current document should respect resist fingerprinting mode on the main thread in OffscreenCanvas.toBlob(); r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D28593
# ==============================================================================

diff -r b9f3ca50c00e -r 904076df6327 dom/canvas/crashtests/crashtests.list
--- a/dom/canvas/crashtests/crashtests.list	Thu Apr 25 07:47:58 2019 +0000
+++ b/dom/canvas/crashtests/crashtests.list	Thu Apr 25 07:47:45 2019 +0000
@@ -52,3 +52,4 @@
 load 1357092.html
 load 1441613.html
 pref(gfx.offscreencanvas.enabled,true) load 1443671.html
+pref(gfx.offscreencanvas.enabled,true) load 1546390.html
